lib/resources/pip.rb

```diff
@@ -15,10 +15,16 @@ class PipPackage < Inspec.resource(1)
       describe pip('Jinja2') do
         it { should be_installed }
       end
+
+      describe pip('django', '/path/to/virtualenv/bin/pip') do
+        it { should be_installed }
+        its('version') { should eq('1.11.4')}
+      end
     "
 
-    def initialize(package_name)
+    def initialize(package_name, virtualenv_path=nil)
```

Comment:
agreed. `pip_path` would make sense. OR having it test for the existence of `@virtualenv_path` + `/bin/pip` before executing.